### PR TITLE
changed loading icon to waiting icon

### PR DESCRIPTION
**Pull request by Alan-AC7**
Fixes https://github.com/getAlby/bitcoin-connect/issues/148

image

### CODE REVIEW

**Comment by rolznz** [comment]
@Alan-AC7 could you also remove the old (unused) icon code?

File: src/components/bc-pay-button.ts [hunk 1]
```diff
@@ -60,7 +60,7 @@ export class PayButton extends withTwind()(BitcoinConnectElement) {
     return html` <div class="inline-flex" @click=${this._onClick}>
       <bci-button variant="primary">
         ${isLoading
-          ? html`<span class="ml-1 mr-1">${loadingIcon}</span>`
+          ? html`${waitingIcon(`w-10 h-10`)}`
```
Comment (rolznz) [inline src/components/bc-pay-button.ts:63]: the margins were removed, does it still match without any jumping when the state changes?

Comment (Alan-AC7) [reply src/components/bc-pay-button.ts:63]: I don't see a difference when the state changes but I'm gonna add them again anyway

**Review by rolznz — APPROVED** [review]
tACK

**Comment by rolznz** [comment]
Thanks @Alan-AC7 ! :rocket:

**Pull request merged**